--- kmldonkey-0.10.1/src/kcmdonkey/kcmdonkey.desktop	2005-07-26 16:40:36.000000000 +0000
+++ kmldonkey-0.10.1.new/src/kcmdonkey/kcmdonkey.desktop	2005-07-26 21:41:55.683403928 +0000
@@ -13,6 +13,7 @@
 Comment[it]=Impostazione dell'integrazione di MLDonkey
 Comment[nl]=MLDonkey-integratie instellen
 Comment[pa]=MLDonkey ਜੋੜ ਸੈਟਅੱਪ
+Comment[pl]=Zintegrowane ustawienia MLDonkey
 Comment[pt]=Configuração da integração do MLDonkey
 Comment[pt_BR]=configuração de integração do MLDonkey
 Comment[ru]=Интеграция MLDonkey 
@@ -37,3 +38,5 @@
 X-KDE-FactoryName=kcmdonkey
 X-KDE-Library=kcmdonkey
 X-KDE-ModuleType=Library
+Categories=Qt;KDE;X-KDE-settings-network;
+# vi: encoding=utf-8
\ No newline at end of file
--- kmldonkey-0.10.1/src/kmldonkey/kmldonkey.desktop	2005-07-26 16:40:55.000000000 +0000
+++ kmldonkey-0.10.1/src/kmldonkey/kmldonkey.desktop	2007-10-16 16:25:09.000000000 +0000
@@ -4,6 +4,8 @@
 Name[hi]=केएमएलडंकी
 Name[sv]=Kmldonkey
 Name[ta]=கேஎம்எல்டாங்கி
+GenericName=An MLDonkey frontend
+GenericName[pl]=Interfejs do MLDonkey
 Exec=kmldonkey %i %m -caption "%c"
 Icon=kmldonkey
 Type=Application
@@ -20,6 +22,7 @@
 Comment[hu]=KDE-alapú kezelőfelület az MLDonkey-hoz
 Comment[it]=Un'interfaccia MLDonkey per KDE
 Comment[nl]=Een MLDonkey-schil voor KDE
+Comment[pl]=Interfejs do MLDonkey dla KDE
 Comment[pt]=Uma interface do MLDonkey para o KDE
 Comment[pt_BR]=Uma interface do MLDonkey para o KDE
 Comment[ru]=Оболочка для MLDonkey в KDE
@@ -27,3 +30,5 @@
 Comment[ta]=கேடீஇகான ஒரு எம்எல்டாங்கியின் முன்னிலை 
 Comment[zh_CN]=一个 KDE 的 MLDonkey 前端
 Terminal=false
+Categories=Qt;KDE;Network;FileTransfer;
+# vi: encoding=utf-8
\ No newline at end of file
